CONTRIBUTING.md

```diff
@@ -13,31 +15,52 @@ Operator Lifecycle Manager is Apache 2.0 licensed and accepts contributions via
 
 ## Reporting bugs and creating issues
 
-Reporting bugs is one of the best ways to contribute. However, a good bug report has some very specific qualities, so please read over our short document on [reporting bugs](./doc/dev/reporting_bugs.md) before submitting a bug report. This document might contain links to known issues, another good reason to take a look there before reporting a bug.
+Reporting bugs is one of the best ways to contribute. However, a good bug report has some very specific qualities, so please read over our short document on [reporting bugs](./doc/dev/reporting_bugs.md) before submitting a bug report. Before filing a bug report, ensure the bug hasn't already been reported by searching through the operator-lifecycle-manager project [Issues][issues].
+
+Any new contribution should be accompanied by a new or existing issue. This issue can help track work, discuss the design and implementation, and help avoid wasted efforts or multiple people working on the same issue, compared to submitting a PR first. Trivial changes, like fixing a typo in the documentation, do not require the creation of a new issue. Proposing larger changes to the operator-lifecycle-manager project may require an enhancement be created in the [operator-framework/enhancements](https://github.com/operator-framework/enhancements/) repository.
```

Comment:
👍

Reply:
> Proposing larger changes to the operator-lifecycle-manager project may require an enhancement be created in the

We should define "larger changes", so there's no ambiguity. Maybe something like: "any change to OLM's behavior, apis, and or testing"?

Reply:
I ran into this same problem locally and ended up just landing on a blanket term/phrase as creating an enhancement feels like a contextual thing. I'm fine with updating to use Nick's phrasing (or if Nick wants to add a commit here changing the existing phrasing) or simply calling out if there's any ambiguity, reach out to the maintainers, either through the list of communication channels listed in the `Communication` header, or create an issue to discuss whether an enhancement is applicable?